airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/leads.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": ["null", "object"],
+  "properties": {
+    "id": {
+      "description": "Unique identifier for the lead",
+      "type": ["null", "string"]
+    },
+    "createdAt": {
+      "description": "Date and time when the lead was created",
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "updatedAt": {
+      "description": "Date and time when the lead was last updated",
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "archived": {
+      "description": "Indicates whether the lead is archived or active",
+      "type": ["null", "boolean"]
+    },
+    "contacts": {
```

Comment:
I couldn't find this field in the documentation
am I missing something?

https://developers.hubspot.com/docs/api/crm/leads

Reply:
This still has to do with the contacts that may be associated with the leads. The associations are flattened as per existing logic. see [here](https://github.com/airbytehq/airbyte/blob/4982562054d8cbfd332261ceffdbcfaa4b1e7dfa/airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py#L1198)